src/server/game/Spells/SpellMgr.cpp

```diff
@@ -4870,7 +4870,119 @@ void SpellMgr::LoadSpellInfoCorrections()
 
     // END OF HALLS OF ORIGINATION SPELLS
 
-    //
+    // SILVERPINE FOREST SPELLS
+
+    ApplySpellFix({
+    83464,  // Chain Visual
+    83467,
+    86803,
+    86805,
+    84112   // Camera spell
+        }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_0].TargetA = SpellImplicitTargetInfo(TARGET_UNIT_TARGET_ANY);
+    });
+
+    ApplySpellFix({
+        83982  // Summon Agatha
+        }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_0].MiscValueB = 3050;
+    });
+
+    ApplySpellFix({
+        85411 // Raise Forsaken
+        }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_0].RadiusEntry = sSpellRadiusStore.LookupEntry(EFFECT_RADIUS_200_YARDS);
+    });
+
+    ApplySpellFix({
+        85600 // Toss Torch
+        }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_0].TargetA = SpellImplicitTargetInfo(TARGET_UNIT_SRC_AREA_ENTRY);
+        spellInfo->Effects[EFFECT_0].RadiusEntry = sSpellRadiusStore.LookupEntry(EFFECT_RADIUS_15_YARDS);
+    });
+
+    ApplySpellFix({
+        85870, // Summon Agatha
+        85869, // Summon Arthura
+        85871, // Summon Daschla
+        85868, // Summon Cromush
+        85864  // Summon Sylvanas
+        }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_0].TargetA = SpellImplicitTargetInfo(TARGET_DEST_DB);
+    });
+
+    ApplySpellFix({
+        83150, // Forsaken Trooper (Male 01)
+        }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_0].MiscValue = 44596;
```

Comment:
why did you change the values of all these transform spells?

Reply:
Apparently, there's a master spell that controls all of these individual spells. However, if left unchanged, the spell chooses a random creature_template ID to modify the creature's appearance, which would lead to inaccuracies, taking into account each individual creature transforms into a set creature. 

So each individual spell was supposed to be set to their connecting creature_template ID.

Reply:
I have removed them, I'll push changes soon. I've just corrected the DB values.